### PR TITLE
added a search bar to the shared watchlist page

### PR DESCRIPTION
**Pull request by chrish0w**

### CODE REVIEW

**Comment by chrish0w** [comment]
added a search bar to the shared watchlist page #197

File: imports/ui/components/lists/ListPopup.jsx
Comment (BrianNge341) [inline imports/ui/components/lists/ListPopup.jsx:1]: should be https://www.thewatchlist.xyz instead of http://localhost:3000

Comment (BrianNge341) [reply imports/ui/components/lists/ListPopup.jsx:1]: line 60 needs to say domain instead of localhost

Comment (BrianNge341) [reply imports/ui/components/lists/ListPopup.jsx:1]: I've made the changes

**Comment by BrianNge341** [comment]
This branch works locally for me

**Review by BrianNge341 — APPROVED** [review]
This branch works locally for me

**Pull request merged**